plugins/modules/aci_l3out_floating_svi_path.py

```diff
@@ -0,0 +1,491 @@
+    node_profile = module.params.get("node_profile")
+    interface_profile = module.params.get("interface_profile")
+    state = module.params.get("state")
+    pod_id = module.params.get("pod_id")
+    node_id = module.params.get("node_id")
+    floating_ip = module.params.get("floating_ip")
+    encap = module.params.get("encap")
+    forged_transmit = module.params.get("forged_transmit").capitalize()
+    mac_change = module.params.get("mac_change").capitalize()
+    promiscuous_mode = module.params.get("promiscuous_mode").capitalize()
+    domain_type = module.params.get("domain_type")
+    domain = module.params.get("domain")
+    enhanced_lag_policy = module.params.get("enhanced_lag_policy")
+    access_encap = module.params.get("access_encap")
+
+    aci = ACIModule(module)
+
+    node_dn = "topology/pod-{0}/node-{1}".format(pod_id, node_id)
+
+    tDn = None
+    if domain_type == "physical":
+        tDn = "uni/phys-{0}".format(domain)
+    elif domain_type == "vmware":
+        tDn = "uni/vmmp-VMware/dom-{0}".format(domain)
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fvTenant",
+            aci_rn="tn-{0}".format(tenant),
+            module_object=tenant,
+            target_filter={"name": tenant},
+        ),
+        subclass_1=dict(
+            aci_class="l3extOut",
+            aci_rn="out-{0}".format(l3out),
+            module_object=l3out,
+            target_filter={"name": l3out},
+        ),
+        subclass_2=dict(
+            aci_class="l3extLNodeP",
+            aci_rn="lnodep-{0}".format(node_profile),
+            module_object=node_profile,
+            target_filter={"name": node_profile},
+        ),
+        subclass_3=dict(
+            aci_class="l3extLIfP",
+            aci_rn="lifp-{0}".format(interface_profile),
+            module_object=interface_profile,
+            target_filter={"name": interface_profile},
+        ),
+        subclass_4=dict(
+            aci_class="l3extVirtualLIfP", aci_rn="vlifp-[{0}]-[{1}]".format(node_dn, encap), module_object=node_dn, target_filter={"nodeDn": node_dn}
+        ),
+        subclass_5=dict(
+            aci_class="l3extRsDynPathAtt",
+            aci_rn="rsdynPathAtt-[{0}]".format(tDn),
+            module_object=tDn,
+            target_filter={"tDn": tDn},
+        ),
+        child_classes=["l3extVirtualLIfPLagPolAtt"],
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        child_configs = []
+        if enhanced_lag_policy is not None and domain_type == "vmware":
+            existing_enhanced_lag_policy = ""
+            if isinstance(aci.existing, list) and len(aci.existing) > 0:
+                for child in aci.existing[0].get("l3extRsDynPathAtt", {}).get("children", {}):
+                    if child.get("l3extVirtualLIfPLagPolAtt"):
+                        existing_enhanced_lag_policy = (
+                            child.get("l3extVirtualLIfPLagPolAtt")
+                            .get("children")[0]
```

Comment:
All the .get should not be chained without a proper returned value as they will create an exception like "AttributeError: 'NoneType' object has no attribute 'get'"

Either use a try/except or pass proper return values.

Reply:
Same for the split at th end.

Reply:
Ok, modified the code to incorporate the comments above.